routes/api.php

```diff
@@ -8,6 +8,8 @@
 Route::get('popular', Api\PopularController::class)->middleware('throttle:60')->name('api.popular-packages');
 Route::get('stats', Api\StatsController::class)->middleware('throttle:60');
 
+Route::get('packages.json', Api\FeedController::class)->middleware('throttle:60');
```

Comment:
It looks like we have the throttle middleware covered in the the `api` middleware group in the http kernel right?

Do you see any issue with removing it here?
```suggestion
Route::get('packages.json', Api\FeedController::class);
```

Reply:
I didn't look at the whole project set-up. I just went with the structure the other API endpoints had, so for that reason I applied the same middleware there. If you want this changed, it should probably be changed for all API endpoints.

Reply:
You should be able to safely remove the throttling on routes with the `throttle:api` in Kernel, see:

https://github.com/laravel/framework/blob/9.x/src/Illuminate/Routing/Middleware/ThrottleRequests.php#L51-L55
and
https://github.com/laravel/framework/blob/470f0dadb35e6f9e80c39ac9051ef6931909ae07/src/Illuminate/Routing/Middleware/ThrottleRequests.php#L82-L104

I didn't remove it from the last commit, as I am not sure the purpose of this PR is to do spring-cleaning 🧹

Reply:
Great points. It doesn't hurt having it there and can be cleaned up in a future PR.